src/backend/utils/storage.js

```diff
@@ -68,5 +68,7 @@ module.exports = {
 
   getPostsCount: () => redis.zcard(POSTS),
 
-  getPost: async guid => redis.hgetall(guid),
+  getPost: guid => {
```

Comment:
You don't need the explicit `return` here:

```js
getPost: guid => redis.hgetall(guid);
```

A fat arrow function will automatically return if you give it an expression vs. a `{...}` body block.